.github/PULL_REQUEST_TEMPLATE

```diff
@@ -29,7 +29,12 @@ Security Update?
 Build Order
 -----------
 
-Please describe in what order maintainers should build this pull request.
+<!-- Please describe in what order maintainers should build this pull request. You can use the following template. -->
+
+```bash
+ciel build <PACKAGES>
```

Comment:
Consider removing `ciel build` form the sequence, as it denotes the command, not the order in which packages are built. It also adds unnecessary clutter in this single-line format.

Reply:
Removed.